apps/webapp/app/send/constants.ts

```diff
@@ -0,0 +1,27 @@
+import { Asset, TabHelper } from './types';
+
+export const tabsHelper: TabHelper = {
+  send: {
+    src: '/funds-gradient.svg',
+    label: 'Sending funds',
+  },
+  receive: {
+    src: '/receive-gradient.svg',
+    label: 'Receiving funds',
+  },
+  ibc: {
+    src: '/ibc-gradient.svg',
+    label: 'IBC funds',
+  },
+};
+
+export const assets: Asset[] = [
+  {
+    name: 'ETH',
+    icon: '/test-asset-icon.svg',
+  },
+  {
+    name: 'BNB',
+    icon: '/test-asset-icon-2.svg',
+  },
+];
```

Comment:
Where did this asset list come from before? Was it hard coded like this?

Reply:
it test data